app/src/components/course/UpdateCourse.jsx

```diff
@@ -0,0 +1,172 @@
+import { useState, useEffect } from "react";
+import { useUpdateCourseContext } from "src/contexts/UpdateCourseContext";
+import { useUserContext } from "src/contexts/UserContext";
+// import { getSkillIdsForCourse, createSkillCourse, deleteSkillCourse } from "src/api/skillCourse";
+import { getSkillsObject } from "src/api/skills";
+// import UpdateCourseSuccess from "./UpdateCourseSuccess";
+import { XMarkIcon } from "@heroicons/react/20/solid";
+import CourseDescription from "./CourseDescription";
+import UpdateConfirmationPopUp from "./UpdateConfirmationPopUp";
+
+export default function HRUpdateCourse() {
+  const { currentUserType } = useUserContext();
+  const { updateCourse } = useUpdateCourseContext();
+
+  const [courseId, setCourseId] = useState(updateCourse.courseId);
```

Comment:
Setting update Course to state is anti-pattern because updateCourse itself is already part of another state

Reply:
How should I resolve this? Because useUpdateCourseContext is required, how do I access useUpdateCourseContext without setting updateCourse to state?